openc3/python/openc3/script/api_shared.py

```diff
@@ -590,47 +588,42 @@ def get_max_output():
     if openc3.script.RUNNING_SCRIPT:
         return openc3.script.RUNNING_SCRIPT.max_output_characters
 
-    ###########################################################################
-    # Scripts Outside of ScriptRunner Support
-    # ScriptRunner overrides these methods to work in the OpenC3 cluster
-    # They are only here to allow for scripts to have a chance to work
-    # unaltered outside of the cluster
-    ###########################################################################
-
-    # TODO:
-    # def start(procedure_name)
-    #   cached = false
-    #   begin
-    #     Kernel::load(procedure_name)
-    #   rescue LoadError => error
-    #     raise LoadError, f"Error loading -- {procedure_name}\n{error.message}"
-    #   end
-    #   # Return whether we had to load and instrument this file, i.e. it was not cached
-    #   !cached
-    # end
-
-    # # Require an additional ruby file
-    # def load_utility(procedure_name)
-    #   return start(procedure_name)
-    # end
-    # def require_utility(procedure_name)
-    #   # Ensure require_utility works like require where you don't need the .rb extension
-    #   if File.extname(procedure_name) != '.rb'
-    #     procedure_name += '.rb'
-    #   end
-    #   @require_utility_cache ||= {}
-    #   if @require_utility_cache[procedure_name]
-    #     return false
-    #   else
-    #     @require_utility_cache[procedure_name] = true
-    #     begin
-    #       return start(procedure_name)
-    #     rescue LoadError
-    #       @require_utility_cache[procedure_name] = false
-    #       raise # reraise the error
-    #     end
-    #   end
-    # end
+
+###########################################################################
+# Scripts Outside of ScriptRunner Support
+# ScriptRunner overrides these methods to work in the OpenC3 cluster
+# They are only here to allow for scripts to have a chance to work
+# unaltered outside of the cluster
+###########################################################################
+
+
+def start(procedure_name):
+    importlib.import_module(procedure_name, None)
+
+
+# Require an additional python file
+def load_utility(procedure_name):
```

Comment:
load_utility and require_utility need to work differently in Python.

If I require_utility or load_utility I should expect to have new methods to call.  (like the collect() utility method from collect.py)

Reply:
supported not support